config/config.go

```diff
@@ -154,6 +155,12 @@ type BigQueryConfig struct {
 	Dataset   string `yaml:"dataset"`
 }
 
+// CloudSourceConfig is used to configure cloudSource integration
+type CloudSourceConfig struct {
+	PrivateKeyPath    string   `yaml:"privateKeyPath"`
```

Comment:
Is there a private key in use to validate incoming requests? Or just copied from `GithubConfig`?

Reply:
Sorry, just copied as placeholder. Will remove.